ui/app/AppLayouts/Chat/CommunityComponents/CommunityUserList.qml

```diff
@@ -83,7 +84,7 @@ Item {
             lastSeen: model.lastSeen
             statusType: model.statusType
             currentTime: root.currentTime
-            offlineColor: "transparent"
+            isOnline: model.online
```

Comment:
make sure this won't break https://github.com/status-im/status-desktop/issues/3163